Routing/Loader/Reader/RestActionReader.php

```diff
@@ -67,6 +67,29 @@ class RestActionReader
     private $availableConventionalActions = ['new', 'edit', 'remove'];
     private $hasMethodPrefix;
 
+    /**
+     * ignore several type hinted arguments.
+     */
+    private $ignoredClasses = [
+        ConstraintViolationListInterface::class,
+        MessageInterface::class,
+        ParamConverter::class,
+        ParamFetcherInterface::class,
+        Request::class,
+        SessionInterface::class,
+        UserInterface::class,
+    ];
+
+    /**
+     * Initializes controller reader.
```

Comment:
all of this is irrelevant. maybe rebase problem.

Reply:
yes, rebasee problem, sorry for that.
resolved...